extensions/wikia/AdEngine3/src/setup.js

```diff
@@ -46,7 +47,13 @@ async function updateWadContext() {
 	}
 }
 
-async function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = true) {
+async function setupAdContext(
+	wikiContext,
+	isOptedIn = false,
```

Comment:
I'd consider creating a single object variable that holds all tracking related flags

Reply:
You right. I don't know why I didn't done this.